src/components/DevelopmentPreview/DevelopmentPreview.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { DevelopmentPreview, DevelopmentPreviewProps } from './DevelopmentPreview';
+import DevelopmentPreviewI18N from "./DevelopmentPreview-i18n.json";
+
+export default {
+    title: "Components/DevelopmentPreview",
+    component: DevelopmentPreview,
+    args: {
+        show: true,
+    },
+    parameters: {
+        i18n: DevelopmentPreviewI18N,
+    },
+} as ComponentMeta<typeof DevelopmentPreview>;
+
+
+const Template: ComponentStory<typeof DevelopmentPreview> = (args) => (
+    <DevelopmentPreview {...args} />
+);
+
+export const showDevPreview = Template.bind({});
+showDevPreview.args = {};
```

Comment:
I think this should show how this looks like with a children, like a sample PageSection or something like that.

It's to understand margins and how the component react at the different breakpoints.